lib/adapters/parsers/csv.js

```diff
@@ -9,6 +9,9 @@ class CSVParser extends base {
     constructor(options) {
         super(options);
         this.separator = options.separator ? options.separator : ',';
+        this.comment = options.comment ? options.comment : null;
```

Comment:
I wonder whether `commentChar` wouldn’t be a better name (it makes it more clear what the value is).